contracts/RewardsSource.sol

```diff
@@ -99,7 +99,7 @@ contract RewardsSource is Governable {
             if (i > _currentSlopeIndex && duration > 0) {
                 nextSlopeIndex = i; // We have moved into a new slope
             }
-            if (slopeEnd < rangeEnd) {
+            if (rangeEnd < slopeEnd) {
```

Comment:
Rather than flipping the sign, I flipped the arguments so that they match the same ordering used through the rest of the function. If I'd done this to start with, the bug probably would have been obvious.